tests/test_layer.py

```diff
@@ -4,7 +4,7 @@
 
 @pytest.fixture(scope="session")
 def custom_crate_a(crate_layer):
-    yield from crate_layer("crate_a", "3.2.x")
+    yield from crate_layer("crate_a", "5.4.x")
```

Comment:
FYI: When using CrateDB 3.2, the test case fails with:

    SystemExit: CrateDB didn't start in time or couldn't form a cluster.
    ERROR: Unrecognized VM option 'UseConcMarkSweepGC'